src/components/Pager/Pager.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import Pager from './Pager';
+
+export default {
+  title: 'Pager',
+  component: Pager,
+  argTypes: {
+    pageSize: { control: 'number' },
+    totalRecords: { control: 'number' },
+  },
+} as ComponentMeta<typeof Pager>;
+
+const Template: ComponentStory<typeof Pager> = (args) => <Pager {...args} />;
+
+export const Primary = Template.bind({});
```

Comment:
What do you think of calling this `Showcase` or `Playground`, the Primary nomenclature was used as an example. 

A pattern that we could follow is to always create a `Playground` or `Showcase` story (your Primary), and then do a few examples, let's say, with different `pageSize` and `totalRecords`.

Stories that could be good for instance:
- Showcase
- 150 items
- 2418 items
- 30 items per page
- And so on...

Kind of thinking the stories as "tests" which will be ultimately "shortcuts" for different cases (different prop values)